ArduPlane/quadplane.cpp

```diff
@@ -769,6 +771,22 @@ void QuadPlane::run_z_controller(void)
     pos_control->update_z_controller();    
 }
 
+/*
+  check if we should relax the attitude controllers
+
+  We relax them whenever we will be using them after a period of
+  inactivity
+ */
+void QuadPlane::check_attitude_relax(void)
+{
+    uint32_t now = AP_HAL::millis();
+    if (now - last_att_control_ms > 500) {
+        attitude_control->relax_attitude_controllers();
+        attitude_control->reset_rate_controller_I_terms();
```

Comment:
This is unneeded as it's implicitly done by `relax_attitude_controllers()`.

Reply:
fixed